apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/create-page/style.scss

```diff
@@ -13,17 +13,17 @@ $create-page-text-color: #a2aab2;
 	min-height: 45px;
 
 	&:focus {
-		color: $white;
-		border: 1px solid $white;
+		color: var(--color-text-inverted);
+		border: 1px solid var(--color-text-inverted);
```

Comment:
Was this reported as an error by stylelint?

Reply:
I didn't see one. Do you see one on your end or do you have concerns about it?